apps/vanilla/src/pages/blackjack/ts/utils/subscriber.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Type for a subscriber that receives updates of type T.
+ */
+export type TSubscriber<T> = {
+
+	/**
+	 * Method to receive a notification of type T from the publisher.
+	 *
+	 * @param {T} message
+	 * @returns {void}
+	 */
```

Comment:
```suggestion
	 * Method to receive a notification of type T from the publisher.
	 * @param message - ?
	 */
```

Is it necessary to mention in the documentation the type of message that will be sent to the method?